experiment/runner.py

```diff
@@ -95,6 +95,8 @@ def _clean_seed_corpus(seed_corpus_dir):
 def get_clusterfuzz_seed_corpus_path(fuzz_target_path):
     """Returns the path of the clusterfuzz seed corpus archive if one exists.
     Otherwise returns None."""
+    if not fuzz_target_path:
+        return None
```

Comment:
Add an error log here because this is unexpected.

Reply:
One question, why is this function even called?

https://github.com/google/fuzzbench/blob/master/experiment/runner.py#L277
I think this is the line that eventually calls this line. But for example, when we observed the error for addison's experiment, the ossfuzz corpus was *NOT* used right? (unless they specified `oss-fuzz-corpus: true`)
then why we would unpack the clusterfuzz seed corpus at all?